src/components/DropdownMenu/DropdownMenu.test.tsx

```diff
@@ -17,14 +17,14 @@ class ResizeObserver {
 }
 window.ResizeObserver = ResizeObserver;
 
-describe('Dropdown', () => {
+describe('Dropdown menu', () => {
   it('Renders a standard dropdown when only triggerText prop is provided', () => {
```

Comment:
some of these statements include menu and others don't. is there a pattern/rationale?